lib/datasets/california-housing.rb

```diff
@@ -0,0 +1,64 @@
+require "csv"
+
+require_relative "dataset"
+require "rubygems/package"
+require "zlib"
+
+module Datasets
+  class CaliforniaHousing < Dataset
+    Record = Struct.new(:longitude,
+                        :latitude,
+                        :housingMedianAge,
+                        :totalRooms,
+                        :totalBedrooms,
+                        :population,
+                        :households,
+                        :medianIncome,
+                        :medianHouseValue)
+
+    def initialize
+      super()
+      @metadata.id = "california-housing"
+      @metadata.name = "California Housing"
+      @metadata.url = "https://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html"
```

Comment:
Can we use http://lib.stat.cmu.edu/datasets/ instead? Because StatLib is the original source.

Reply:
Ok. This source seems to have been down, but can update to use it.  It may be helpful to indicate secondary download locations when they are available.

Reply:
Done. Had used the wrong url.